federation/test/extensions.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+
+import Decimal from 'decimal.js-light'
```

Comment:
I am not sure that we need this feature in the federation. This code provides `expect.decimalEqual()` for the unit tests. Do we have to test equality of decimals in the federation?

Reply:
At the moment the federation contains as graphql-request only the query { test { api } }.
The next version of federation will contain the graphql-query { getPublicKey { publicKey } }.
In future versions to support X-Community-TXs we will have for sure Decimals in the graphql-queries.

So should i remove this file for this version now and add it again as soon as we need it or could it be as part of the test-suite?